packages/prisma/schema.prisma

```diff
@@ -204,8 +204,9 @@ model BookingReference {
   meetingId       String?
   meetingPassword String?
   meetingUrl      String?
-  booking         Booking? @relation(fields: [bookingId], references: [id])
+  booking         Booking? @relation(fields: [bookingId], references: [id], onDelete: Cascade)
```

Comment:
@zomars What you think about this?
Error occurs when trying to delete booking because payment it's involved.

Reply:
If it works it works